compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -1181,12 +1182,16 @@ impl<'a> PartialEvaluator<'a> {
             );
             None
         };
-        let (args, ctls_arg) = self.resolve_args(
-            (store_item_id.package, callable_decl.input).into(),
-            args_value.clone(),
-            ctls,
-            fixed_args,
-        );
+        let (args, ctls_arg) = self
+            .resolve_args(
+                (store_item_id.package, callable_decl.input).into(),
+                args_value.clone(),
+                ctls,
+                fixed_args,
+            )
+            .map_err(|()| {
+                Error::EvaluationFailed("qubis in invocation are not unique".to_string(), args_span)
```

Comment:
Qubits

Reply:
Fixed by relying on the same string from evaluator.